Microsoft.Toolkit.Uwp.UI.Media/Brushes/AcrylicBrush.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Toolkit.Uwp.UI.Media.Brushes.Base;
+using Microsoft.Toolkit.Uwp.UI.Media.Pipelines;
+using Windows.UI;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Brushes
```

Comment:
```suggestion
namespace Microsoft.Toolkit.Uwp.UI.Media
```

We didn't add Brushes on top of the namespace as we were keeping consistency with the older `System.Windows.Media` namespace from the RadialGradientBrush.  I mean it's the only thing in the package currently, though I see us moving non-brush things in the future.  I still think it's fine to have it in the main space.

Thoughts?

Reply:
I personally don't mind that, and I could make a commit to move all of those into just the `.Media` namespace then. The only note though is that we also have other public namespaces though, such as [`.Pipelines](https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/f3a4a3df76ea4dffc35fd0e9fe5c116370eb3f3c/Microsoft.Toolkit.Uwp.UI.Media/Pipelines) now. We can do either of the following:

- Leave all the brushes in just `.Media` for simplicity and ignore the inconsistency
- Move all the brushes in `.Media` to get consistency with the other namespaces (and possibly with new ones being added in the future).

Personally I don't mind having the explicit `.Brushes` namespace, but I can see how having just `.Media` there would be maybe more intuitive for users. Maybe we should get some more feedbacks from others too? I'd be leaning towards a uniform `.Brushes` for now I think 🤔